frontend/src/views/instructions.js

```diff
@@ -0,0 +1,31 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+import React from "react";
+import airplane from "../images/airplane.png";
+import { Button } from "../components/ui";
+import { Link } from "@reach/router";
+
+function Instructions() {
+  return (
+    <div>
+      <img
+        src={airplane}
+        css={{ margin: "auto", display: "block", marginBottom: 20 }}
+        alt="airplane"
+      />
+      <div css={{ textAlign: "center" }}>
+        <h1>Welcome</h1>
+        <p css={{ marginBottom: 24 }}>
+          Today you have to program the next 7 flights. Please be sure to make
```

Comment:
Typos: "Please **make** sure to do it well" "decisions" 
And that last sentence is a bit off. I'd put it like this: "Your decisions could make a lot of money for the company, but it could also make the company lose lots of money. Choose carefully." or something like that